### ISSUE TITLE
Demo File

### ISSUE THREAD

**Issue by prescod** [trinketapp/pygal.js#7]
Can someone help me figure out how to make a decent demo file? It would make it dramatically easier to test and supply patches.

Below is my first attempt which results in

> "Failed to load resource: net::ERR_FILE_NOT_FOUND : deps.js"

Is a build step required? Here's my HTML:

```
<html>
<head>
<script src="bower_components/jquery/dist/jquery.js"></script>
<script src="bower_components/skulpt/skulpt.js"></script>
<script>
// the domOutput is called whenever the chart is rendered
// and is expected to append the provided html to the DOM
// and return the resulting jquery element
Sk.domOutput = function(html) {
  return $('body').append(html).children().last();
};

// tell Skulpt where to find pygal.js and its dependencies
Sk.externalLibraries = {
  pygal : {
    path : './__init__.js',
    dependencies : [
      'bower_components/highcharts/highcharts.js',
      'bower_components/highcharts/highcharts-more.js'
    ]
  }
};

// optionally configure the size (in pixels) at which the charts should render
Sk.availableWidth = 600;
Sk.availableHeight = 400;
</script>
</head>
<body>

</body>
</html>
```

**Comment by brianpmarks** [comment]
@prescod While not a minimal example, the link below might help out. It's an HTML trinket that sets up skulpt, highcharts, and pygal. The `__init__.js` included in the trinket is currently the same as the one in this repo. You could theoretically make changes to it within the trinket for testing... Of course you can't open a pull request from the trinket but hopefully it'll help.

https://trinket.io/embed/html/4642c05cb4

As far as the missing `deps.js` file goes, I think you get that when using the non-minified version of skulpt. If you want to pull in the latest build we use you can pull it from [this repo](https://github.com/trinketapp/skulpt-dist).

Anyway, let me know of any questions.

**Comment by brianpmarks** [comment]
Fixed by #8.
